packages/relay/src/Environment.js

```diff
@@ -46,50 +19,26 @@ export default function createEnvironment(
     networkHeaders.authorization = accessToken;
   }
 
-  const fetchQuery = async (
-    operation,
-    variables,
-  ): Promise<{| data: Object, errors?: $ReadOnlyArray<Object> |}> => {
+  const fetchQuery = async (operation, variables): Promise<FetcherResponse> => {
     // TODO: fetch persisted queries instead (based on operation.id)
```

Comment:
Are we ever going to to this, or is it time to remove this comment?

Reply:
I hope so, yes - I am still waiting for it... 🙂